### PR TITLE
Import GPG key for the repository

### PR DESCRIPTION
**Pull request by albertodonato**
Import the GPG key used to sign the ESM repository.

Currently, the public key is the one for staging.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 flake8
+mock
```
Comment (ericsnowcurrently) [inline requirements.txt]: If this project were Python 3, you'd have mock for free in the stdlib. :)

Comment (albertodonato) [inline requirements.txt]: I know, but precise is quite old, and this file will be included in an existing package, so we don't want to add dependencies (python3)

**Comment by albertodonato** [comment]
[Write script to configure the ESM repo on machines](https://trello.com/c/l89ynCFo/242-write-script-to-configure-the-esm-repo-on-machines)

**Comment by blackboxsw** [comment]
Should be remove the apt-key upon disable?

**Comment by albertodonato** [comment]
@blackboxsw I think it's fine to leave it there. It won't do any harm, and removing the repo is not something the customer would normally do (since you'd end up with no updates at all after EOL)

**Pull request merged**